.github/workflows/build-packages.yml

```diff
@@ -20,14 +20,14 @@ jobs:
 
       - name: Install dependencies
         run: |
-          npm ci
-          npm ci --prefix src/packages/excalidraw
-          npm ci --prefix src/packages/utils
+          yarn
```

Comment:
The equivelant of `npm ci` is [`yarn --frozen-lockfile` ](https://classic.yarnpkg.com/en/docs/cli/install/#toc-yarn-install-frozen-lockfile). Otherwise, deps won't be reproducible.